quad-py/src/lib.rs

```diff
@@ -40,14 +40,14 @@ fn qag_vec(
         if epsabs.is_some() {
             epsabs.unwrap()
         } else {
-            1e-4
+            1.49e-8
```

Comment:
Isn't it a weird number?

Reply:
Is the same used in scipy.integrate.quad : https://docs.scipy.org/doc/scipy/reference/generated/scipy.integrate.quad.html .